shoes-core/spec/shoes/shared_examples/dsl/rgb.rb

```diff
@@ -5,7 +5,8 @@
   let(:alpha) { 133 } # cornflowerblue
 
   it "sends args to Shoes::Color" do
-    expect(Shoes::Color).to receive(:new).with(red, green, blue, alpha)
+    expect(Shoes::Color).to receive(:new).ordered
```

Comment:
This test is failing now because when the app gets created it calls `Shoes::Color` to setup the system color and this expectation gets fired.

The solution I found online is to use `ordered` like this so that you say: I want the second time it gets called to be the following.

The only trouble is that the shoes specs (with a Mock backend) only call `Shoes::Color` one time so it fails this way too. Anyone know the right way to handle this?